casper/src/test/scala/coop/rchain/casper/util/rholang/RuntimeManagerTest.scala

```diff
@@ -176,6 +187,19 @@ class RuntimeManagerTest extends FlatSpec with Matchers {
     }
   }
 
+  "closeBlock" should "make epoch change and reward validator" in effectTest {
+    runtimeManagerResource.use { runtimeManager =>
+      compareSuccessfulSystemDeploys(runtimeManager)(genesis.body.state.postStateHash)(
+        new CloseBlockDeploy(
+          initialRand = Blake2b512Random(Array(0.toByte))
+        ),
+        new CloseBlockDeploy(
+          initialRand = Blake2b512Random(Array(0.toByte))
+        )
+      )(_ => true)
+    }
+  }
+
```

Comment:
Can you please add a test which shows failed execution of close block. For now just a seed mismatch should be fine. It would be nice to have more complicated cases with other failures in PoS.